### ISSUE TITLE
CAN interface for imx6 board(ARM cortex A9 quad)

### ISSUE THREAD

**Issue by tejadeep** [rbei-etas/busmaster#1227]
i need to connect imx6 board to busmaster tool in my PC, can you please suggest me any interface hardware is available,

**Comment by yegorich** [comment]
Have you already looked at the official hardware support [list](https://github.com/rbei-etas/busmaster/wiki/Hardware-support)?

**Comment by tejadeep** [comment]
yeah i searched, but i didn't found anything related to imx6 board can you please suggest me

**Comment by tejadeep** [comment]
Hi Yegorich, Actually we need to control all peripherals in imx6 board through CAN controller, for that we are using Busmaster tool, but we are struck while interfacing busmaster and imx6 board
